.github/workflows/README.md

```diff
@@ -113,21 +113,21 @@ on github, or with the `gh` command line tool. Parameters:
 
 ## Making a release
 
-1. Ensure that changelog is up-to-date and version in `main` matches the version you want to release (otherwise see step 3).
+1. Check that changelog is up-to-date and the `VERSION` file on the `main` branch matches the version you want to release. If not, see step 3.
```

Comment:
Maybe talk this over in person? I might be thinking about this the wrong way, but unless the plan is to start updating the changelog incrementally (which isn't impossible, but in my experience leads to merge conflicts), updating the changelog needs to be at the start of the release process, not the end.

Reply:
The auto-number is nice, but it does mean that if a step is inserted, "3" might be pointing at the right thing... but my preference is really to just get rid of this branch in the flow chart, and this would be moot.

Reply:
Follow-up from discussion: 
- If it's a patch release, no change to the version number should be needed: We incremented the patch after the previous release.
- If it's a minor release, `python tools/channel_tool.py bump_version --position minor`
- Similarly, if it's a major release, `python tools/channel_tool.py bump_version --position major`